src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -52,6 +57,22 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
 
+        if (argMultimap.getValue(PREFIX_DEPARTMENT).isPresent()) {
+            editPersonDescriptor.setDepartment(
+                    ParserUtil.parseDepartment(argMultimap.getValue(PREFIX_DEPARTMENT).get())
```

Comment:
Perhaps split these nested function calls into multiple lines to make it clearer?